src/pages/ViewFreaks/ViewFreaks.jsx

```diff
@@ -25,29 +23,23 @@ function ViewFreaks() {
   if (!data) return <p>Not found</p>;
 
   const freaks = data.freaks.nodes;
-  const { technologies, projects } = data;
+  const filters = (({ technologies, projects }) => ({ technologies, projects }))(data);
```

Comment:
What is the idea here? Extract only two properties of one object?

Reply:
Right, this is redundant. I removed this line and use the properties directly